### PR TITLE
[DOCS-2287] add RUM watchdog insights docs page

### PR DESCRIPTION
**Pull request by hdelaby**
### What does this PR do?

* Create page for "RUM Watchdog Insights" doc

### Motivation

* New feature! 🎉 

### Preview

https://docs-staging.datadoghq.com/hdelaby/rum_watchdog_insights/real_user_monitoring/explorer/watchdog_insights/

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/real_user_monitoring/explorer/watchdog_insights.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+  * The timeseries for RUM errors that have the field.
+
+{{< img src="real_user_monitoring/explorer/watchdog_insights/error_outlier_side_panel.png" alt="Error Outlier side panel" style="width:60%;" >}}
+
+### Latency outliers
+
+A latency outliers insight shows fields (that is, [faceted tags or attributes][1]) that are associated with performance bottlenecks, matching the current query. This casts light on `key:value` pairs that are having worse performance than the baseline, allowing you to uncover performance bottlenecks among a subset of real users.
+
+Latency outliers are computed for [Core Web Vitals][2] (First Contentful Paint, First Input Delay and Cumulative Layout Shift) and [Loading Time][3].
```
Comment (alai97) [inline content/en/real_user_monitoring/explorer/watchdog_insights.md:70]:
```suggestion
Latency outliers are computed for [Core Web Vitals][2] such as First Contentful Paint, First Input Delay, and Cumulative Layout Shift, and [Loading Time][3].
```

File: content/en/real_user_monitoring/explorer/watchdog_insights.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+* The **banner card** version of the insight shows:
+
+  * The name of the field.
+  * The value of the performance metric that has this field, and what the baseline is for the rest of the data.
```
Comment (alai97) [inline content/en/real_user_monitoring/explorer/watchdog_insights.md:75]:
```suggestion
In the **banner card** view, you can see:

* The field name.
* The performance metric value containing the field and the baseline for the rest of the data.
```

File: content/en/real_user_monitoring/explorer/watchdog_insights.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+{{< img src="real_user_monitoring/explorer/watchdog_insights/latency_outlier_s_card.png" alt="Latency Outlier card (S)" style="width:60%;" >}}
```
Comment (alai97) [inline content/en/real_user_monitoring/explorer/watchdog_insights.md:77]:
```suggestion
{{< img src="real_user_monitoring/explorer/watchdog_insights/latency_outlier_s_card.png" alt="Latency Outlier banner card view" style="width:100%;" >}}
```

File: content/en/real_user_monitoring/explorer/watchdog_insights.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+
+* The **side panel card** of the insight shows, in addition:
+
+  * The timeseries of the performance metric for this field, and of the baseline for the rest of the data.
```
Comment (alai97) [inline content/en/real_user_monitoring/explorer/watchdog_insights.md:81]:
```suggestion
In the **side panel card** view, you can see a timeseries of the performance metric for the field and baseline for the rest of the data.
```

File: content/en/real_user_monitoring/explorer/watchdog_insights.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+
+{{< img src="real_user_monitoring/explorer/watchdog_insights/latency_outlier_l_card.png" alt="Latency Outlier card (L)" style="width:60%;" >}}
```
Comment (alai97) [inline content/en/real_user_monitoring/explorer/watchdog_insights.md:83]:
```suggestion
{{< img src="real_user_monitoring/explorer/watchdog_insights/latency_outlier_l_card.png" alt="Latency Outlier side panel view" style="width:100%;" >}}
```

File: content/en/real_user_monitoring/explorer/watchdog_insights.md [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+
+* The **full side panel** of the card shows, in addition:
+
+  * A list of RUM views that have this field, to be able to find the root cause of the performance issue in the [performance waterfall][4].
```
Comment (alai97) [inline content/en/real_user_monitoring/explorer/watchdog_insights.md:87]:
```suggestion
In the **full side panel** view, you can see a list of RUM events that contain the field. Look for the root cause of the performance issue in the [performance waterfall][4].
```

File: content/en/real_user_monitoring/explorer/watchdog_insights.md [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+
+{{< img src="real_user_monitoring/explorer/watchdog_insights/latency_outlier_side_panel.png" alt="Latency Outlier side panel" style="width:60%;" >}}
```
Comment (alai97) [inline content/en/real_user_monitoring/explorer/watchdog_insights.md:89]:
```suggestion
{{< img src="real_user_monitoring/explorer/watchdog_insights/latency_outlier_side_panel.png" alt="Latency Outlier full side panel view" style="width:100%;" >}}
```

**Review by alai97 — CHANGES_REQUESTED** [review]
Standardized image alt text, restructured lists in the Collections section, and suggested comments with doc feedback.

File: content/en/real_user_monitoring/explorer/watchdog_insights.md [hunk 8]
```diff
@@ -0,0 +1,98 @@
...
+
+## Overview
+
+With Watchdog Insights, find your way faster to the root cause of problems with contextual insights you receive as you explore Real User Monitoring (RUM) events. Watchdog Insights complements your expertise and instincts, giving nudges toward where you could focus your troubleshooting attention. 
+
+<div class="alert alert-warning">
+Watchdog Insights for RUM is a beta feature, rolling out to customers using RUM. If you have feedback, contact <a href="https://docs.datadoghq.com/help">Datadog support</a>.
+</div>
+
+In the following example, Watchog Insights highlights that the instance of our application deployed on `view.url_host:www.shopist.io` causes most of the errors observed in a specific time range.
```
[the remark this answers is not in the document]
Comment (hdelaby) [reply content/en/real_user_monitoring/explorer/watchdog_insights.md:25]: 🙈

**Comment by hdelaby** [comment]
Hey @alai97, thanks a lot for the review! While I'm OK with most of the comments, I mostly copied/pasted https://docs.datadoghq.com/logs/explorer/watchdog_insights/ and adapted it to RUM. Does it change the way we want to do things here?

**Comment by alai97** [comment]
Hey @hdelaby, thanks for the additional context. It doesn't change my doc review of the Watchdog Insights for RUM; in fact, I recommend porting over the changes here to that page for Watchdog Insights for Logs to be more consistent. Let me know if you're down with that and either you and I can create a separate PR for that page 😀

File: content/en/real_user_monitoring/explorer/watchdog_insights.md [hunk 9]
```diff
@@ -0,0 +1,98 @@
...
+  tag: "Blog"
+  text: "Monitor Core Web Vitals with RUM"
+- link: "https://www.datadoghq.com/blog/datadog-mobile-rum/"
+  tag: "Blog"
+  text: "Improve mobile user experience with Datadog Mobile RUM"
+---
+
+## Overview
+
+With Watchdog Insights, find your way faster to the root cause of problems with contextual insights you receive as you explore Real User Monitoring (RUM) events. Watchdog Insights complements your expertise and instincts, giving nudges toward where you could focus your troubleshooting attention.
```
[the remark this answers is not in the document]
Comment (hdelaby) [reply content/en/real_user_monitoring/explorer/watchdog_insights.md:19]: I think this changes the meaning of the sentence too much. Watchdog Insights helps you navigating through the RUM Explorer by recommending outliers and potential performance bottlenecks impacting a subset of users.

The suggested change implies that RM offers contextual insights AND watchdog insights - which reads a bit weird.

**Comment by hdelaby** [comment]
Hey @alai97 thanks a lot for your work on this one, it's not an easy concept and you nailed it!

File: content/en/real_user_monitoring/explorer/watchdog_insights.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (alai97) [reply content/en/real_user_monitoring/explorer/watchdog_insights.md:19]: Good feedback, @hdelaby! I amended my suggestion, how's this for a rewording?

**Comment by alai97** [comment]
Thanks, Hugo! One more comment and we're good to merge.

**Comment by hdelaby** [comment]
We're good to go! Thank you 💯

**Review by alai97 — APPROVED** [review]
Sounds good, @hdelaby! I'll create a ticket to port over these changes into Watchdog Insights for Logs.

**Pull request merged**